sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/ArtifactHelper.java

```diff
@@ -0,0 +1,76 @@
+package org.openmrs.maven.plugins.utility;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.openmrs.maven.plugins.model.Artifact;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.twdata.maven.mojoexecutor.MojoExecutor;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.twdata.maven.mojoexecutor.MojoExecutor.artifactId;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executeMojo;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executionEnvironment;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.goal;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.groupId;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.name;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.plugin;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.version;
+
+/**
+ * The purpose of this class is to handle all interactions with Maven that require retrieving artifacts from the Maven repository
+ */
+public class ArtifactHelper {
```

Comment:
One of my goals that led me to doing this in the first place was to start identifying functionality that we might refactor into common libraries.  This is the start of that effort, and the integration test that corresponds with it is necessary to make sure it is operating as expected.